processors/baggage/baggagetrace/processor.go

```diff
@@ -7,48 +7,27 @@ import (
 	"context"
 
 	"go.opentelemetry.io/otel/attribute"
-	"go.opentelemetry.io/otel/baggage"
+	otelbaggage "go.opentelemetry.io/otel/baggage"
 	"go.opentelemetry.io/otel/sdk/trace"
 )
 
-// Filter returns true if the baggage member should be added to a span.
-type Filter func(member baggage.Member) bool
-
-// AllowAllMembers allows all baggage members to be added to a span.
-var AllowAllMembers Filter = func(baggage.Member) bool { return true }
-
-// SpanProcessor is a [trace.SpanProcessor] implementation that adds baggage
-// members onto a span as attributes.
-type SpanProcessor struct {
-	filter Filter
-}
+// SpanProcessor is a processing pipeline for spans in the trace signal.
+type SpanProcessor struct{}
 
 var _ trace.SpanProcessor = (*SpanProcessor)(nil)
 
-// New returns a new [SpanProcessor].
+// New returns a new SpanProcessor.
 //
 // The Baggage span processor duplicates onto a span the attributes found
 // in Baggage in the parent context at the moment the span is started.
-// The passed filter determines which baggage members are added to the span.
-//
-// If filter is nil, all baggage members will be added.
-func New(filter Filter) *SpanProcessor {
-	return &SpanProcessor{
-		filter: filter,
-	}
+func New() trace.SpanProcessor {
```

Comment:
Should we just return `baggagecopy.NewSpanProcessor`?

Reply:
Done.